### PR TITLE
Combine CI/CD docker compose.

### PR DESCRIPTION
**Pull request by ZePan110**
## Description

Combine CI/CD docker compose.

## Issues

List the issue or RFC link this PR is working on. If there is no such link, please mark it as `n/a`.

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)
- [x] Others (enhancement, documentation, validation, etc.)

## Dependencies

List the newly introduced 3rd party dependency if exists.

## Tests

Describe the tests that you ran to verify your changes.

### CODE REVIEW

File: .github/workflows/_comps-workflow.yml [hunk 1]
```diff
@@ -49,16 +49,12 @@ jobs:
...
-          else
-            docker_compose_yml=${{ github.workspace }}/.github/workflows/docker/compose/${{ inputs.service }}-compose.yaml
-          fi
+          docker_compose_yml=${{ github.workspace }}/.github/workflows/docker/compose/${{ inputs.service }}-compose.yaml
           echo "docker_compose_path=${docker_compose_yml}" >> $GITHUB_OUTPUT
           if [ -f "$docker_compose_yml" ]; then
             echo "file_exists=true" >> $GITHUB_OUTPUT
           else
-            echo "There is no ${{ inputs.mode }} part of ${{ inputs.service }} that needs to be executed."
+            echo "The docker_compose_yml does not exist!"
```
Comment (chensuyue) [inline .github/workflows/_comps-workflow.yml:57]: `docker_compose_yml ` -> `$docker_compose_yml `

File: .github/workflows/_comps-workflow.yml [hunk 2]
```diff
@@ -67,6 +63,17 @@ jobs:
               git clone https://github.com/vllm-project/vllm.git vllm-openvino
               cd ./vllm-openvino && git checkout v0.6.1 && cd ../
           fi
+          if [[ $(grep -c "vllm-hpu:" ${docker_compose_yml}) != 0 ]]; then
+               git clone https://github.com/HabanaAI/vllm-fork.git vllm-fork
```
Comment (chensuyue) [inline .github/workflows/_comps-workflow.yml:67]: add commit print.

File: .github/workflows/_comps-workflow.yml [hunk 3]
```diff
@@ -67,6 +63,17 @@ jobs:
               git clone https://github.com/vllm-project/vllm.git vllm-openvino
               cd ./vllm-openvino && git checkout v0.6.1 && cd ../
```
Comment (chensuyue) [inline .github/workflows/_comps-workflow.yml:64]: add commit print.

**Review by chensuyue — APPROVED** [review]
Remove the invalid comments in yaml.

**Review by lvliang-intel — APPROVED** [review]

**Review by lkk12014402 — APPROVED** [review]

**Pull request merged**